src/assets/css/UserReservations.css

```diff
@@ -1,30 +1,26 @@
-* {
+.container {
   margin: 0;
-  padding: 0;
-  box-sizing: border-box;
-}
```

Comment:
Very good job using classes and avoiding selectors that may cause problems.
This will help you a lot during your journey as a developer, and for that reason, I mentioned it because it is the knowledge that will help you become a better developer. 

Excellent!!!!!